FSharp-7.0/FS-1124-interfaces-with-static-abstract-members.md

```diff
@@ -57,6 +57,10 @@ type C() =
 
 Static abstract methods may not be declared in classes.
 
+Static abstract methods may not be called directly on interfaces.
```

Comment:
It should be explicitly mentioned that virtual methods can be called directly on the interfaces.

Reply:
Done. Thanks for the review